packages/calcite-components/src/components/stepper-item/stepper-item.e2e.ts

```diff
@@ -36,4 +38,39 @@ describe("calcite-stepper-item", () => {
     await stepperItem.click();
     expect(stepperItemSelect).toHaveReceivedEventTimes(1);
   });
+
+  const stepperItemHTML = html`
+    <calcite-stepper-item heading="Confirm and complete" icon numbered description="Some subtext">
+      <calcite-notice open width="full">
```

Comment:
Is `notice` needed for the test? If not, let's drop it or replace it for simple text.